src/components/posts/PostList.js

```diff
@@ -0,0 +1,21 @@
+import PostItem from "./PostItem";
+import classes from "./PostList.module.css"
+
+function PostList(props) {
+  return (
+    <ul className={classes.list}>
+      {props.posts.map((post) => (
```

Comment:
Nice use of a map here to get all of the posts displayed

Reply:
Thank you, I am something of a genius